### ISSUE TITLE
Plenary sessions not present

### ISSUE THREAD

**Issue by russel** [abingham/accu-2017-elm-app#31]
The keynotes, the lightning talks, and the ACCU AGM are not listed, only the parallel sessions are present. This may be a development choice for this stage, and that is fine. Going forward though, it would be good to have them.

**Comment by abingham** [comment]
Yes, I kept things simple for now. I don't think I even serve up those kinds of talks right now.

**Comment by russel** [comment]
Let's put that up as an enhancement for later, possibly much later. :-)
